plaso/cli/storage_media_tool.py

```diff
@@ -53,13 +53,26 @@ class StorageMediaTool(tools.CLITool):
   _UNITS_1000 = ['B', 'kB', 'MB', 'GB', 'TB', 'EB', 'ZB', 'YB']
   _UNITS_1024 = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'EiB', 'ZiB', 'YiB']
 
-  _USER_PROMPT_VSS = '\n'.join(textwrap.wrap((
+  _USER_PROMPT_APFS = (
+      'Please specify the identifier(s) of the APFS volume that should be '
+      'processed: Note that a range of volumes can be defined as: 3..5. '
+      'Multiple volumes can be defined as: 1,3,5 (a list of comma separated '
+      'values). Ranges and lists can also be combined as: 1,3..5. The first '
+      'volume is 1. All volumes can be defined as "all". If no volumes are '
+      'specified none will be processed. You can abort with Ctrl^C.')
+
+  _USER_PROMPT_TSK = (
+      'Please specify the identifier of the partition that should be '
+      'processed. All partitions can be defined as: "all". Note that you can '
+      'abort with Ctrl^C.')
+
+  _USER_PROMPT_VSS = (
       'Please specify the identifier(s) of the VSS that should be processed: '
       'Note that a range of stores can be defined as: 3..5. Multiple stores '
       'can be defined as: 1,3,5 (a list of comma separated values). Ranges '
       'and lists can also be combined as: 1,3..5. The first store is 1. All '
       'stores can be defined as "all". If no stores are specified none will '
-      'be processed. You can abort with Ctrl^C.'), 79))
+      'be processed. You can abort with Ctrl^C.')
```

Comment:
These prompts have proven confusing for users. Please add a prompt on an extra line similar to "VSS Identifiers:" so it's the clear that Plaso is waiting for user input.

Reply:
Ack, yeah now you mention it I recall that we had a conversion about this. I'll also make these change in dfVFS.